common/script/mapping-tables.js

```diff
@@ -175,9 +149,9 @@ function mappingTables() {
 
       // create content for each <details> element; add row header's content to summary
       var details = document.createElement('details');
-      details.className = 'map removeOnSave';
+      details.className = 'map';
 
-      var detailsHTML = '<summary id="' + id + '">' + summary;
+      var detailsHTML = '<summary id="' + id + '" data-id="' + id + '">' + summary;
```

Comment:
adding an `id` and a `data-id` with identical values seems strange. What is the reasoning here? Maybe we should add a comment here in the code so the next person touching this code knows what is going on.

Reply:
We don't need them both at the same time! I just added a commit to make it so only one or the other exists. When converting between the "table view" and "summary view", the hidden view now has only the data-id attribute and the visible view has only an id attribute.

Does the make it less confusing?